### DISCUSSION TITLE
Extend Wololo API

### DISCUSSION THREAD

**Discussion by YouHaveTrouble** [PurpurMC/Purpur#436] [__Dismissed__] [answered]
Extend Evoker.wololo() to be able to target any entity or block and maybe add an event that would allow us to modify the result such as changing entity type, block type or data etc.

**Comment by BillyGalbreath** [comment]
Testing webhook

**Comment by BillyGalbreath** [comment, marked as answer]
Going to go ahead and close this one for the reasons I said in discord the other day. The more I think about it, the more I'm pretty sure its going to break someone's workflow to make these changes built into the server jar. Best that a plugin does it.

> i'm looking at your wololo request YouHaveTrouble and i'm thinking it would be easier to make that into a plugin, tbh. i fear changing the internals from Sheep to Entity would break ABI and someone will end up screaming at us :3 And to make it work with Blocks would need a completely new Goal, or a major rewrite of the existing one
with Paper's Mob Goal API it is entirely possible to do it without any NMS at all, even :wink:
